Common/Securities/SecurityExchangeHours.cs

```diff
@@ -196,7 +196,7 @@ public DateTime GetNextMarketOpen(DateTime localDateTime, bool extendedMarket)
             do
             {
                 var marketHours = GetMarketHours(time.DayOfWeek);
-                if (!marketHours.IsClosedAllDay && !_holidays.Contains(time.Ticks))
+                if (!marketHours.IsClosedAllDay && !_holidays.Contains(time.Date.Ticks))
```

Comment:
Interesting. This is a great change, just curious how it just became an issue? Presumably all consumers of this method were at one point only sending in the `DateTime.Date` value, but this is definitely more correct. Perhaps now that this change is here, we should check consumers to see if they're doing the `.Date` and remove it (why do it twice)?